roles/kubernetes/node/tasks/main.yml

```diff
@@ -32,6 +32,13 @@
   tags:
     - kubelet
 
+- name: Make sure dynamic kubelet configuration directory is writeable
+  file:
+    path: "{{ dynamic_kubelet_configuration_dir }}"
+    mode: 0700
```

Comment:
My mistake. 0700 makes no sense on a directory. It should be 0600

Reply:
done